.travis.yml

```diff
@@ -1,16 +1,15 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.4.2
+  - 2.6.4
```

Comment:
Can we test it on Ruby 2.5 as well? I think that it makes sense at least until we upgrade all our apps to Ruby 2.6